playbooks/cluster_setup_debian.yaml

```diff
@@ -6,6 +6,21 @@
 # with an iso created by build_debian_iso
 # This playbooks can also configure VMs created with build_debian_iso
 
+- name: Check if we are using a correct seapath flavor
+  hosts:
+    - cluster_machines
+    - standalone_machine
+  tasks:
+    - name: Retrieve Seapath system version
+      command: "uname -a"
+      register: "uname_output"
+
+    - name: Check if current Ansible branch is compatible
+      fail:
+        msg: "You are trying to configure with a Seapath flavor incompatible with the current Ansible git branch"
```

Comment:
I would be more straight forward in the message :
"The current branch (debian-main) is incompatible with the Yocto version of SEAPATH. Please use the main branch to configure this hypervisor"

Reply:
Done